COLLABORATOR_GUIDE.md

```diff
@@ -391,6 +391,28 @@ Runtime Deprecations and End-of-life APIs (internal or public) must be
 handled as semver-major changes unless there is TSC consensus to land the
 deprecation as a semver-minor.
 
+Possible reasons for Runtime deprecations can be:
+
+* Node.js will no longer be able to support the API, for example, if required
+  code is being removed in an upstream project.
+
+* The API is deemed too hard or impossible to use correctly, increasing the
+  risk of undetected bugs or security issues in existing code.
+
+* Breaking the API makes way for simplification of its implementation that is
+  significant enough to make maintainability of Node.js easier, or enables
+  implementation of features that would otherwise be impossible.
+
+In addition, a Documentation-Only deprecation may be issued if it is desirable
+to recommend a single canonical way out of multiple different ones
+which achieve a particular goal with Node.js.
+
+A Runtime deprecation should not be added to an API that
+requires only trivial maintenance. A Runtime deprecation should
+not be added simply because an API has never been documented
+(or has only accidentally been exposed) if there is significant usage
+of the API in the ecosystem.
```

Comment:
Can this please be changed to e.g.:

```
In case there is significant usage of an API in the ecosystem a Runtime deprecation should
not be added simply because the API has never been documented (or has only accidentally
been exposed) without existing alternative. The deprecation notice has to reference the
alternative in such a case.
```

And I think it would be best to move this up below the last entry of `Possible reasons for Runtime deprecations can be:`.